template.env

```diff
@@ -1,8 +1,16 @@
-DATABASE_URL=postgres://joplin@127.0.0.1:5433/joplin
-JANIS_URL=https://janis.austintexas.io
+# Postgres vars
+JOPLIN_POSTGRES_PASSWORD='choose_your_own'
+DATABASE_URL=postgres://joplin:${JOPLIN_POSTGRES_PASSWORD}@127.0.0.1:5433/joplin
+
+# Django vars
+DEBUG=1
+DEBUG_TOOLBAR=0
 MONITOR_PERFORMANCE=False
-DEPLOYMENT_MODE=LOCAL
-DJANGO_LOG_LEVEL=INFO
+DJANGO_LOG_LEVEL=DEBUG
 DJANGO_SECRET_KEY=dummy
```

Comment:
do we know what it would take to change this to 'choose_your_own' as well? Would it break things?

Reply:
Nope, you can change DJANGO_LOG_LEVEL, it wouldn't break anything. 
`template.env` is just a template for your own local `.env` file. You can set whichever values you'd prefer in your .env.

Reply:
But you're right. DJANGO_LOG_LEVEL=INFO is a better default value.

Reply:
Oh, you meant DJANGO_SECRET_KEY. Yeah, should be a choose_your_own.